src/Components/IssueCard.jsx

```diff
@@ -311,13 +311,18 @@ const useStyles = makeStyles((theme) => ({
     cursor: 'pointer',
   },
   body: {
-    height: (props) => props.expandText ? 'auto' : '62px',
-    margin: '5px',
-    paddingLeft: '5px',
-    overflow: 'hidden',
-    textOverflow: 'ellipsis',
-    fontSize: '1em',
-    lineHeight: '1.3em',
+    'height': (props) => props.expandText ? 'auto' : '62px',
```

Comment:
<img width="321" alt="image" src="https://user-images.githubusercontent.com/3433606/181225483-81019ec0-88b2-4a4d-857e-e06ce12a4eed.png">

something has shifted with the markdown perhaps adjust 62px to 58px -- perhaps?

Reply:
Done